### ISSUE TITLE
NameError: name 'unicode' is not defined

### ISSUE THREAD

**Issue by kazuki-ma** [o19s/elyzer#15]
```
$ elyzer --es foo --index bar --analyzer my_analizer 'xxx'
Traceback (most recent call last):
  File "/usr/bin/elyzer", line 9, in <module>
    load_entry_point('elyzer==1.1.0', 'console_scripts', 'elyzer')()
  File "/usr/lib/python3.4/site-packages/elyzer/__main__.py", line 42, in main
    es=es))
  File "/usr/lib/python3.4/site-packages/elyzer/elyzer.py", line 36, in getAnalyzer
    normalizeAnalyzer(analyzer)
  File "/usr/lib/python3.4/site-packages/elyzer/elyzer.py", line 19, in normalizeAnalyzer
    if (isinstance(analyzer['char_filter'], str) or isinstance(analyzer['char_filter'], unicode)):
NameError: name 'unicode' is not defined
```

Do not you have people of the same problem?

**Comment by kazuki-ma** [comment]
My patch... Is it right thing?

```diff
def normalizeAnalyzer(analyzer):
    try:
-        if (isinstance(analyzer['char_filter'], str) or isinstance(analyzer['char_filter'], unicode)):
+        if (isinstance(analyzer['char_filter'], str)):
            analyzer['char_filter'] = [analyzer['char_filter']]
    except KeyError:
        analyzer['char_filter'] = []
```

**Comment by simonhughes22** [comment]
Having same issue. Is this a python 3.5 or 3.6 thing?

**Comment by sstults** [comment]
Fixed in v1.2.0
